jwt.go

```diff
@@ -84,13 +274,20 @@ func (t *Token) SigningString() (string, error) {
 // keyFunc will receive the parsed token and should return the key for validating.
 // If everything is kosher, err will be nil
 func Parse(tokenString string, keyFunc Keyfunc) (*Token, error) {
+	return ParseWithClaims(tokenString, keyFunc, &Claims{})
```

Comment:
By default the Parse function will use the structured object, if other claim types are wanted you can directly call ParseWithClaims.